src/test/regress/expected/multi_explain_0.out

```diff
@@ -0,0 +1,334 @@
+             Relation Name: "pg_merge_job_0061"                    +
+             Alias: "pg_merge_job_0061"
+(1 row)
+
+-- Test verbose
+EXPLAIN (COSTS FALSE, VERBOSE TRUE)
+	SELECT sum(l_quantity) / avg(l_quantity) FROM lineitem;
+                                                                                                               QUERY PLAN                                                                                                               
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ Distributed Query into pg_merge_job_0062
+   Task Count: 6
+   Tasks Shown: Sample
+   ->  Task
+         Node: host=localhost port=57637 dbname=regression
+         ->  Aggregate
+               Output: sum(l_quantity), sum(l_quantity), count(l_quantity)
+               ->  Seq Scan on public.lineitem_102010 lineitem
+                     Output: l_orderkey, l_partkey, l_suppkey, l_linenumber, l_quantity, l_extendedprice, l_discount, l_tax, l_returnflag, l_linestatus, l_shipdate, l_commitdate, l_receiptdate, l_shipinstruct, l_shipmode, l_comment
+ 
+ Master Query
+   ->  Aggregate
+         Output: (sum(intermediate_column_62_0) / (sum(intermediate_column_62_1) / sum(intermediate_column_62_2)))
+         ->  Seq Scan on pg_temp_2.pg_merge_job_0062
+               Output: intermediate_column_62_0, intermediate_column_62_1, intermediate_column_62_2
+(15 rows)
+
+-- Test join
+EXPLAIN (COSTS FALSE)
+	SELECT * FROM lineitem
+	JOIN orders ON l_orderkey = o_orderkey AND l_quantity < 5
+	ORDER BY l_quantity DESC LIMIT 10;
+                                   QUERY PLAN                                   
+--------------------------------------------------------------------------------
+ Distributed Query into pg_merge_job_0063
+   Task Count: 6
+   Tasks Shown: Sample
+   ->  Task
+         Node: host=localhost port=57637 dbname=regression
+         ->  Limit
+               ->  Sort
+                     Sort Key: lineitem.l_quantity
+                     ->  Hash Join
+                           Hash Cond: (lineitem.l_orderkey = orders.o_orderkey)
+                           ->  Seq Scan on lineitem_102010 lineitem
+                                 Filter: (l_quantity < 5::numeric)
+                           ->  Hash
+                                 ->  Seq Scan on orders_102015 orders
+ 
+ Master Query
+   ->  Limit
+         ->  Sort
+               Sort Key: intermediate_column_63_4
+               ->  Seq Scan on pg_merge_job_0063
+(20 rows)
+
+-- Test all tasks output
+SET citus.explain_all_tasks TO on;
+EXPLAIN (COSTS FALSE)
+	SELECT avg(l_linenumber) FROM lineitem WHERE l_orderkey > 9030;
+                        QUERY PLAN                         
+-----------------------------------------------------------
+ Distributed Query into pg_merge_job_0064
```

Comment:
I'm not sure we can realistically address this right now - and perhaps it doesn't make sense as we want to get rid of the temp tables anyway - but I think displaying the temp table name here is a bad idea. Will just introduce pointless variance, and it's not really giving the user more information.